subprojects/tooling-api/src/main/java/org/gradle/tooling/Configurable.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.tooling;
+
+import org.gradle.api.Incubating;
+
+/**
+ * Same as java.util.function.Consumer, but the Tooling API still needs to be compatible with Java 6.
+ *
+ * @param <T> the target type
+ * @since 7.5
+ */
+@Incubating
+public interface Configurable<T> {
```

Comment:
I think we can use `Action` in the TAPI, or at least we have in `org.gradle.tooling.BuildController`

Reply:
Configurable is gone now; I've replaced it with `Action`.